Gruntfile.js

```diff
@@ -66,7 +66,24 @@ module.exports = function (grunt) {
                     protocol: 'https'
                 }
             }
-        }
+        },
```

Comment:
As discussed offline make sure to squash all the commits into a single commit before merging.

Reply:
I disabled Allow merge commits (Add all commits from the head branch to the base branch with a merge commit.) and allowed "Allow squash merging "(Combine all commits from the head branch into a single commit in the base branch.) in https://github.com/aws/connect-rtc-js/settings to make sure I'm not pushing commits separately